apps/src/code-studio/progressRedux.ts

```diff
@@ -453,21 +455,6 @@ const userProgressFromServer = (
   });
 };
 
-/**
```

Comment:
Some minor cleanup here: previously, we were fetching the current level directly out of `state.progress.lessons` without performing `processLevel` like we do in some other places (in `progressReduxSelectors` for example). This confused me a bit at first because there are some slight differences in the level structure before and after processing; before, the level's `id` is the script level ID, while the level ID is stored in its `ids` array and/or `activeId` field. After processing, we assign the `id` to the `activeId` if present. To be consistent with the rest of progressRedux and just to make things more clear, I opted to use a function from `progressReduxSelectors` instead which will perform `processLevel`, and add an explicit `scriptLevelId` field to the processed level, so we know which ID we're dealing with.